### PR TITLE
Automatic check if an object already exists on ENA

### PR DESCRIPTION
**Pull request by bedroesb**
- [x] Incorporating the module of Ignacio's wrapper in the CLI code
- [x] Exporting the tsv tables also in --draft mode 
- [x] Further testing
- [x] Documentation update
- [x] auto fill the action specified in the command

This will be the last feature to be added before the improvements in error handling and logging will be integrated.

### CODE REVIEW

**Review by bgruening — COMMENTED** [review]
Cool!

File: ena_upload/ena_upload.py [hunk 1]
```diff
@@ -94,10 +97,32 @@ def check_columns(df, schema, action):
...
+                if auto_action:
+                    for index, row in df.iterrows():
+                        remote_present = np.nan
+                        try:
+                            remote_present = str(identify_action(
+                                schema, str(df['alias'][index]), dev)).upper()
+
+                        except Exception as e:
+                            print(e)
+                            print(
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply ena_upload/ena_upload.py:109]: I totally agree! Will be my next PR if everything goes well :)

File: ena_upload/ena_upload.py [hunk 2]
```diff
@@ -953,18 +983,23 @@ def main():
...
                                                    schema_targets,
                                                    action)
-            # save updates in new tables
-            save_update(schema_tables, schema_dataframe)
+        else:
+            schema_dataframe = update_table(schema_dataframe,
+                                            schema_targets,
+                                            schema_update)
+        # save updates in new tables
+        save_update(schema_tables, schema_dataframe)
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply ena_upload/ena_upload.py]: You are right!

**Comment by bedroesb** [comment]
@bgruening sorry for bothering! Do you think this PR is ready for merge? We would like to make a screencast soon and this is one last part :)

**Comment by bgruening** [comment]
Please don't let me stop you ;)

**Comment by bedroesb** [comment]
@bgruening you are not! Thanks for checking so quickly!

**Pull request merged**